deebot_client/commands/json/water_info.py

```diff
@@ -34,7 +34,7 @@ def _handle_body_data_dict(
             mop_attached = bool(mop_attached)
 
         event_bus.notify(
-            WaterInfoEvent(WaterAmount(int(data["amount"])), mop_attached=mop_attached)
+            WaterInfoEvent(WaterAmount(int(data["amount"])), SweepType(int(data["sweepType"])), mop_attached=mop_attached)
```

Comment:
This will not work as not all bot will have a `sweepType` in the response. Please change it so it is optional